test/service-worker.test.js

```diff
@@ -0,0 +1,34 @@
+const webdriver = require('selenium-webdriver');
+const { until, By } = require('selenium-webdriver');
+const { Options } = require('selenium-webdriver/firefox');
+
+const assert = require('assert');
+
+/*
+
+Test Auth functionalities (Login, Signup, Logout) with selenium and mocha
+
+*/
+
+
+describe('Service Worker Tests', function() {
```

Comment:
what did you have in mind for service worker tests? I think you can delete this file for now - let's just finish the user account tests for now